timeseries/tests/unittests/test_predictor.py

```diff
@@ -149,11 +150,12 @@ def test_given_hyperparameters_and_quantiles_when_predictor_called_then_model_ca
     predictor.fit(
         train_data=DUMMY_TS_DATAFRAME,
         hyperparameters=hyperparameters,
-        tuning_data=DUMMY_TS_DATAFRAME,
+        tuning_data=tuning_data,
```

Comment:
This test did not catch the bug introduced in #5178 because 1/ it did not check the values of the predictions and 2/ `tuning_data=DUMMY_TS_DATAFRAME` disabled the `MultiWindowBacktestingModel`  usage.